Exiled.API/Features/Items/Firearm.cs

```diff
@@ -185,7 +185,7 @@ public float FireRate
         }
 
         /// <summary>
-        /// Gets or sets the recoil settings of the firearm, if it's an automatic weapon.
+        /// Gets or sets the recoil settings of the firearm if it's an automatic weapon.
```

Comment:
```suggestion
        /// Gets or sets the recoil settings of the firearm, if it's an automatic weapon.
```
Comma is appropriate

Reply:
Added